examples/README.md

```diff
@@ -0,0 +1,9 @@
+# Venafi Ansible Role examples
+
+In this section you'll find some use case scenarios that implement Venafi Ansible Role to automate SSL certificate generation and installation on your infrastructure.
+
```

Comment:
Because I'm not the subject matter expert on Ansible, I might be missing something...so is Ansible also about securing app delivery using, in this case, the Venafi Ansible Role with either F5 or Citrix ADCs? If that's right, I suggest mirroring more the Terraform intro para by stating these explicitly. Of course automating cert gen/install is the Venafi value-add, but the real value that customers care most about is what that buys them in the end, right? e.g. securing app delivery, etc. Anyway, my two cents.

Reply:
Changes have been performed